ftest.sh

```diff
@@ -64,9 +64,50 @@ restore_dist_files() {
 
 }
 
+cleanup() {
+    restore_dist_files "${yaml_files[@]}"
+    i=5
+    while [ $i -gt 0 ]; do
+        pdsh -l ${REMOTE_ACCT:-jenkins} -R ssh -S \
```

Comment:
(lint) Double quote to prevent globbing and word splitting. [SC2086]

Reply:
is this just asking for "${REMOTE_ACCT:-jenkins}" ?   does that break it?